plugins/auth-node/src/sign-in/commonSignInResolvers.ts

```diff
@@ -38,11 +43,28 @@ export namespace commonSignInResolvers {
             );
           }
 
-          return ctx.signInWithCatalogUser({
-            filter: {
-              'spec.profile.email': profile.email,
-            },
-          });
+          try {
+            return await ctx.signInWithCatalogUser({
+              filter: {
+                'spec.profile.email': profile.email,
+              },
+            });
+          } catch (err) {
```

Comment:
Probably best to only do this if `err.name === 'NotFoundError'`

Reply:
Indeed, makes sense, have updated the PR